db/migrate/20220915141751_create_long_read_qc_result.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# Added a new table long_read_qc_result to be able to query QC data from the extraction process
+class CreateLongReadQcResult < ActiveRecord::Migration[6.0]
+  def change
+    create_table :long_read_qc_result do |t|
+      t.string :labware_barcode, null: false
+      t.string :sample_id, null: false
+      t.string :assay_type, null: false
+      t.string :assay_type_key, null: false
+      t.string :units
+      t.string :value, null: false
+      t.string :id_lims
+      t.string :id_long_read_qc_result_lims
+      t.datetime :created
+      t.datetime :last_updated
+      t.datetime :recorded_at
+    end
+  end
+end
```

Comment:
Can we add descriptions to the fields, please? Something I really like about the MLWH is that it generally has descriptions for the fields in the database (unlike most of our other apps). Happy to help come up with them if you like.